bot/tests/test_reporter_mail.py

```diff
@@ -143,4 +147,4 @@ def _check_email(request):
     conf = {"emails": ["test@mozilla.com"]}
     r = BuildErrorsReporter(conf)
 
-    r.publish(mock_issues, mock_revision)
+    r.publish(mock_coverity_issues, mock_revision)
```

Comment:
:+1:

Reply:
Add an assert on the `logger.info` about the email address used using [pytest-structlog](https://pypi.org/project/pytest-structlog/). The library is already used in the unit tests

Reply:
This is so nice, asserting the log!